CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [5.7.0] - 2022-02-08
```

Comment:
I'd suggest keeping as **Unreleased**, considering the PR is from a feature branch and the merge is to develop.

Reply:
oh I missed that 👍

Reply:
you bet, will do